### PR TITLE
Fixes admiral recall

### LINKED ISSUE #27787: Admiral recall doesn't actually do anything
It gives an admin message that the admin sent the recall, but the recall is never actually triggered
@coiax

### PR DESCRIPTION
**Pull request by Shadowlight213**
Fixes #27787 

@ChangelingRain You removed a bit more than the mess~

### CODE REVIEW

**Comment by optimumtact** [comment]
kick me would you eh

**Comment by kevinz000** [comment]
WTF METAGRUDGE

**Comment by lzimann** [comment]
My fault, meant to remove the timer and probably forgot to readd the proc cal. Speaking of which, get rid of this nonsense timer

**Comment by Shadowlight213** [comment]
@lzimann done

File: code/controllers/subsystem/shuttle.dm [hunk 1]
```diff
@@ -221,7 +221,7 @@ SUBSYSTEM_DEF(shuttle)
 /datum/controller/subsystem/shuttle/proc/centcom_recall(old_timer, admiral_message)
 	if(emergency.mode != SHUTTLE_CALL || emergency.timer != old_timer)
 		return
-	emergency.cancel(/area/centcom)
+	emergency.cancel(new /area/centcom)
```
Comment (Cyberboss) [inline code/controllers/subsystem/shuttle.dm]: Use locate(/area/centcom) in GLOB.sortedAreas

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/controllers/subsystem/shuttle.dm (hunk 1 above)
Comment (Shadowlight213) [inline code/controllers/subsystem/shuttle.dm]: @Cyberboss there is no actual centcom area. There's only the children. I checked when debugging this.

Comment (optimumtact) [inline code/controllers/subsystem/shuttle.dm]: I would just leave it null

Comment (AnturK) [inline code/controllers/subsystem/shuttle.dm]: Please change this, creating empty areas actually has side-effects so avoid this please.

**Pull request merged**